circuits_benchmark/transformers/acdc_circuit_builder.py

```diff
@@ -1,85 +1,163 @@
 from acdc.TLACDCCorrespondence import TLACDCCorrespondence
 from circuits_benchmark.transformers.circuit import Circuit
 from circuits_benchmark.transformers.circuit_node import CircuitNode
+from acdc.TLACDCEdge import EdgeType
 
 
 def build_acdc_circuit(corr: TLACDCCorrespondence) -> Circuit:
-  circuit = Circuit()
-
-  for (child_name, child_index, parent_name, parent_index), edge in corr.edge_dict().items():
-    if edge.present:
-      parent_head_index = None
-      if (parent_index is not None and
-          len(parent_index.hashable_tuple) > 2 and
-          parent_index.hashable_tuple[2] is not None):
-        parent_head_index = parent_index.hashable_tuple[2]
-
-      child_head_index = None
-      if (child_index is not None and
-          len(child_index.hashable_tuple) > 2 and
-          child_index.hashable_tuple[2] is not None):
-        child_head_index = child_index.hashable_tuple[2]
-
-      from_node = CircuitNode(parent_name, parent_head_index)
-      to_node = CircuitNode(child_name, child_head_index)
-      circuit.add_edge(from_node, to_node)
+    circuit = Circuit()
+
+    for (child_name, child_index, parent_name, parent_index), edge in corr.edge_dict().items():
+        if edge.present and edge.edge_type != EdgeType.PLACEHOLDER:
+            parent_head_index = None
+            if (
+                parent_index is not None
+                and len(parent_index.hashable_tuple) > 2
+                and parent_index.hashable_tuple[2] is not None
+            ):
+                parent_head_index = parent_index.hashable_tuple[2]
+
+            child_head_index = None
+            if (
+                child_index is not None
+                and len(child_index.hashable_tuple) > 2
+                and child_index.hashable_tuple[2] is not None
+            ):
+                child_head_index = child_index.hashable_tuple[2]
+
+            from_node = CircuitNode(parent_name, parent_head_index)
+            to_node = CircuitNode(child_name, child_head_index)
+            circuit.add_edge(from_node, to_node)
 
-  return circuit
+    return circuit
 
 
 def get_full_acdc_circuit(n_layers: int, n_heads: int) -> Circuit:
```

Comment:
Fix this.

Reply:
`full_corr = TLACDCCorrespondence.setup_from_model(tl_model, use_pos_embed=use_pos_embed)        full_circuit = build_acdc_circuit(full_corr)`